nntrainer/layers/conv2d_layer.cpp

```diff
@@ -404,9 +407,13 @@ void Conv2DLayer::forwarding(RunLayerContext &context, bool training) {
   }
 
   filter_kernel.reshape(filter_dim);
-  status = hidden_.add_i(bias_kernel);
-  if (status != ML_ERROR_NONE) {
-    throw std::invalid_argument("[Conv2D] adding bias failed");
+  if (auto &disable_bias = std::get<props::DisableBias>(*layer_impl_props);
+      disable_bias.empty() || disable_bias.get() == false) {
+    Tensor &bias_kernel = context.getWeight(wt_idx[ConvParams::bias]);
+    status = hidden_.add_i(bias_kernel);
+    if (status != ML_ERROR_NONE) {
+      throw std::invalid_argument("[Conv2D] adding bias failed");
+    }
```

Comment:
I know you are just transpiling but want to ask for the opinion.
should this be added for fc as well(it's missing)? or as we assures dimension matches, we don't check this?

Reply:
it's not needed. `add_i` always returns ML_ERROR_NONE from tensor.cpp. Even this can be removed but I will just leave it here.

Reply:
Cool